src/utils/graph/solver.js

```diff
@@ -20,8 +20,8 @@ import { Solver, Variable } from 'kiwi.js';
  */
 export const solveLoose = (constraints, iterations, constants) => {
   for (let i = 0; i < iterations; i += 1) {
-    for (const co of constraints) {
-      co.base.solve(co, constants);
+    for (const constrain of constraints) {
```

Comment:
```suggestion
    for (const constraint of constraints) {
```
Need to change the one below too